src/bindings/python/flux/job.py

```diff
@@ -258,8 +258,16 @@ def job_list(flux_handle, max_entries=1000, attrs=[], userid=os.geteuid(), state
     return JobListRPC(flux_handle, "job-info.list", payload)
 
 
-def job_list_inactive(flux_handle, since=0.0, max_entries=1000, attrs=[]):
-    payload = {"since": since, "max_entries": max_entries, "attrs": attrs}
+def job_list_inactive(flux_handle, since=0.0, max_entries=1000, attrs=[], name=None):
+    if name:
+        payload = {
+            "since": since,
+            "max_entries": max_entries,
+            "attrs": attrs,
+            "name": name,
+        }
+    else:
+        payload = {"since": since, "max_entries": max_entries, "attrs": attrs}
```

Comment:
Minor nit. Feel free to take or leave this, but could be shorter as:
```suggestion
    payload = {"since": since, "max_entries": max_entries, "attrs": attrs}
    if name:
        payload['name'] = name
```

Reply:
Oh, I like this.  Thanks!